tools/generate_patches.sh

```diff
@@ -0,0 +1,137 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/usr/bin/env bash
```
Bash may not always be under /bin

Reply:
Done